test/config_parser_SUITE.erl

```diff
@@ -260,13 +307,34 @@ loglevel(_Config) ->
     ?err(parse_host_config(#{<<"general">> => #{<<"loglevel">> => <<"debug">>}})).
 
 hosts(_Config) ->
-    ?eq([#config{key = hosts, value = [<<"host1">>, <<"host2">>]}],
-        parse(#{<<"general">> => #{<<"hosts">> => [<<"host1">>, <<"host2">>]}})),
+    ?eq([#config{key = hosts, value = [<<"host1">>]}],
+        parse(#{<<"general">> => #{<<"hosts">> => [<<"host1">>]}})),
+    ?eq_list([#config{key = hosts, value = [<<"host1">>, <<"host2">>]},
```

Comment:
Just FYI: there is a function `compare_config` that is meant for this purpose - it compares two lists of config records with using `compare_ordered_lists` and `compare_unordered_lists` when needed.